tests/unit/lms/services/canvas_api/client_test.py

```diff
@@ -12,12 +12,11 @@
 from lms.services.canvas_api.client import CanvasAPIClient
 from tests import factories
 
+pytestmark = pytest.mark.usefixtures("http_session", "oauth_token")
 
-class TestCanvasAPIClientGetToken:
-    # This is the only test where we fake out the underlying class, because
-    # this _one_ call is just a pass through.
 
-    def test_get_token(self, canvas_api_client, authenticated_client):
+class TestGetToken:
```

Comment:
This is our naming style for test classes:

* If the class-under-test is small enough to cover with a single test class, it's just `TestFooClass`
* If the class-under-test is large, then it's one `TestBarMethod` class for each `FooClass.bar_method()`